[cimg] build error on x86-windows-static & x64-windows-static

Package: cimg:x86-windows-static@3.6.5#1

**Host Environment**

- Host: x64-windows
- Compiler: MSVC 19.44.35222.0
- CMake Version: 3.31.10
-    vcpkg-tool version: 2025-12-05-d84fb805651077f4ed3a97b61fbad2b513462f8b
    vcpkg-scripts version: d245228101 2025-12-18 (19 hours ago)

**To Reproduce**

`vcpkg install cimg:x86-windows-static`

**Failure logs**

```
Downloading https://github.com/GreycLab/CImg/archive/v.3.6.5.tar.gz -> GreycLab-CImg-v.3.6.5.tar.gz
GreycLab-CImg-v.3.6.5.tar.gz.7060.part: error: download from https://github.com/GreycLab/CImg/archive/v.3.6.5.tar.gz had an unexpected hash
note: Expected: 7a0a66cf8dfb9923be0ee576f5dc6c53a8704a92ce9b1b3b8d1a7c6b5a1d94f231fce25a813f0fb6e6afd0f75111a8efb91f3e05c57621900ef588e00bf8d06c
note: Actual  : 4580ea45df9650df82dc81f444078cfbee5a21dcef4fa59b6c2bae7d531f3bb89318814ba3bdc0ec1dea7aaab678c7e6ab12d8e61afa3492d657922d482ae5ab
CMake Error at scripts/cmake/vcpkg_download_distfile.cmake:136 (message):
  Download failed, halting portfile.
Call Stack (most recent call first):
  scripts/cmake/vcpkg_from_github.cmake:120 (vcpkg_download_distfile)
  ports/cimg/portfile.cmake:3 (vcpkg_from_github)
  scripts/ports.cmake:206 (include)



```

--- a/ports/cimg/portfile.cmake
+++ b/ports/cimg/portfile.cmake
@@ -2,8 +2,9 @@ set(VCPKG_BUILD_TYPE release) # header-only
 
 vcpkg_from_github(OUT_SOURCE_PATH SOURCE_PATH
     REPO GreycLab/CImg
-    REF "v.${VERSION}"
-    SHA512 7a0a66cf8dfb9923be0ee576f5dc6c53a8704a92ce9b1b3b8d1a7c6b5a1d94f231fce25a813f0fb6e6afd0f75111a8efb91f3e05c57621900ef588e00bf8d06c
+    # Using commit id becuase upstream likes to change tags
+    REF 50674b2dcfe004673f629bbb37bc9c960f2ae614
+    SHA512 efdeab32fa4378818bb599c34590dcebef2b790594eacb822bb9eba9679c73b59f37c75b3ea2e348ea6cfa47866a72402dfc660a14b930514a7452d9a0bbc8ea
     HEAD_REF master
 )
 

--- a/ports/cimg/vcpkg.json
+++ b/ports/cimg/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "cimg",
   "version": "3.6.5",
-  "port-version": 1,
+  "port-version": 2,
   "description": "The CImg Library is a small, open-source, and modern C++ toolkit for image processing",
   "homepage": "https://github.com/GreycLab/CImg",
   "license": "CECILL-C AND CECILL-2.0",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -1738,7 +1738,7 @@
     },
     "cimg": {
       "baseline": "3.6.5",
-      "port-version": 1
+      "port-version": 2
     },
     "cinatra": {
       "baseline": "0.9.7",

--- a/versions/c-/cimg.json
+++ b/versions/c-/cimg.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "9fc45e348f2f2582f83d7b46bab8ed6ea83512a0",
+      "version": "3.6.5",
+      "port-version": 2
+    },
+    {
       "git-tree": "3fce3d2b620c8eec111d885d3f134cbd22ee11b4",
       "version": "3.6.5",
       "port-version": 1

